.circleci/config.yml

```diff
@@ -32,8 +32,37 @@ jobs:
           paths:
             - .circleci/continue_config.yml
 
+  # Bump version and create release branch
+  generate-release:
+    working_directory: ~/audius-client
+    docker:
+      - image: circleci/node:14.18
+    steps:
+      - checkout
+      - run:
+          name: Bump version
+          command: |
+            git config --global user.email "audius-infra@audius.co"
+            git config --global user.name "audius-infra"
+            PREV_VERSION=$(grep -m1 version ./lerna.json | awk -F: '{ print $2 }' | sed 's/[", ]//g')
```

Comment:
My vote would be to use `jq` here, but this works!

If you can send me a sample lerna.json I can write the `jq` command. :)

Reply:
Oh cool, I pulled this from an existing command but would love to make it more elegant! The `lerna.json` is super simple:

```
{
  "packages": [
    "packages/*"
  ],
  "version": "1.1.11"
}
```

Reply:
Ah perfect!

I would rewrite your line like this:

```
PREV_VERSION=$(jq -r .version ./lerna.json)
```

`jq .` shows the whole json, `jq .version` only shows the version key, `jq -r` strips the quotes using raw formatting, and the input can either be a filename or piped directly to `jq` via `| jq`.

I hope that helps!

Reply:
Here are the related docs if you ever want to go heavy into `jq` (it takes TIME and you quickly forget things :sweat_smile: ) or ever want to do something similar for yaml using `yq`:

* https://stedolan.github.io/jq/manual/
* https://mikefarah.gitbook.io/yq/

They're definitely nice tools when trying to extract data. :nerd_face:

Reply:
sorry don't mean to butt in but aren't jq and yq not included by default? also up to @sliptype but i feel like this might just be a bit over optimization to try to install and use it for such a small syntactic gain 🙂

Reply:
I'm not sure if `jq` is included by default, but it *should* be since it's a goto like `grep` or `awk` when messing with json files. If not, `jq` is [easily installable](https://stedolan.github.io/jq/download/).

We also don't need `yq`. Sebastian sounded excited about a more elegant solution, so I was just sharing `yq` for his own future reference.

Indeed, as I mentioned in my original post, the current solution works and usage of `jq` is optional (even if it is the ideal choice when interacting with json files).

Reply:
I'll see if it's included in the image we are running! Installing it seems like overkill in this situation

Reply:
+1 on both counts! :)